### PR TITLE
WIP : update to block syndromes calculation in qrcode.cpp

### PR DESCRIPTION
**Pull request by ZhengQiushi**
```
force_builders_only=linux,docs
```

### CODE REVIEW

**Comment by alalek** [comment]
Please stop creating of new Pull Requests.
Keep all your work in a single one (with all history of reviews).

**Comment by alalek** [comment]
Re-opened as #17812 is broken by fork removal

**Comment by ZhengQiushi** [comment]
> Keep all your work in a single one (with all history of reviews).

sorry , I'm not familiar  with PR so... I will stop creating new PR

File: modules/objdetect/src/qrcode.cpp [hunk 1]
```diff
@@ -5,2354 +5,3432 @@
...
-    CV_TRACE_FUNCTION();
-    vector<Vec3d> result;
-    const int height_bin_barcode = bin_barcode.rows;
-    const int width_bin_barcode  = bin_barcode.cols;
-    const size_t test_lines_size = 5;
-    double test_lines[test_lines_size];
-    vector<size_t> pixels_position;
-
-    for (int y = 0; y < height_bin_barcode; y++)
+    vector<Vec3d> QRDetect::searchHorizontalLines()
```
Comment (dkurt) [inline modules/objdetect/src/qrcode.cpp:744]: Please keep origin alignment

**Pull request closed without merging**